pages/book/cells.mdx

```diff
@@ -113,11 +112,42 @@ contract SerializationExample {
 }
 ```
 
-### `remaining` [#serialization-bytes64]
+### `remaining` [#serialization-remaining]
+
+To illustrate what `remaining{:tact}` serialization type does, let's take a look at [cell manipulation primitives](#cells-immutability) and their [TL-B][tlb] representation produced by Tact:
+
+```tact
+contract SerializationExample {
+    // By default
+    cRef: Cell;    // ^cell in TL-B
+    bRef: Builder; // ^builder in TL-B
+    sRef: Slice;   // ^slice in TL-B
+
+    // With `remaining`
+    cRem: Cell as remaining;    // remainder<cell> in TL-B
+    bRem: Builder as remaining; // remainder<builder> in TL-B
+    sRem: Slice as remaining;   // remainder<slice> in TL-B
+}
+```
+
+There, `^cell`, `^builder` and `^slice` in [TL-B][tlb] syntax mean the reference to [`Cell{:tact}`](#cells), [`Builder{:tact}`](#builders) and [`Slice{:tact}`](#slices) values respectively, while the `remainder<…>` of `cell`, `builder` or `slice` tells that the given value would be stored as a `Slice{:tact}` directly and not as a reference.
+
+To draw parallels with [cell manipulation instructions](#cells-immutability), specifying `remaining{:tact}` is like using [`Builder.storeSlice(){:tact}`][b-5] and [`Slice.loadSlice(){:tact}`][s-5] instead of [`Builder.storeRef(){:tact}`][b-8] and [`Slice.loadRef(){:tact}`][s-8], which are to be used by default.
```

Comment:
Let's take a _concrete example_ such as

```
message TransferMessage {
    signature: Slice as bytes64; // you might want to change this one or go ahead and explain `as bytesX` and `as remaning` in this section in one PR
    transfer: Slice as remaining;
}
```

and show the equivalent code demonstrating how the message cell would be parsed for this case and for the case without these `as` annotations

Reply:
Hmm, I don't think adding `bytesX` into the description of `remaining` is a good idea.

On other note, do you suggest to show the concrete Struct/Message, and then describe what will happen whenever that message is received or we assign a Slice to, say,  `self.transfer` and show what would happen under the hood? I kinda did that in previous iteration, and it quickly got too convoluted, so I thought this nice and short paragraph with conceptual load/store references is enough.

We could always iterate on that later if the community in Tact chat would find that a bad or misleading comparison :)

Reply:
I'm suggesting to show how to parse a message with fields annotated `as remaining`, `as bytesX`. We could add also how to build a message like that as well, although I think parsing is more important to show and it already explains how things are structured under the hood. Code is better than text, code + text is better than code. TL-B schemes are good but only to the users who already understand all those concepts.